src/layouts/helpers.tsx

```diff
@@ -72,3 +72,11 @@ export function conditionallyDisplay(
 export function capitalize(value: string): string {
   return value.charAt(0).toUpperCase() + value.slice(1);
 }
+
+/* Check if string is a valid UUID */
+export function checkIfValidUuid(input: string):boolean {
+  // 8-4-4-4-12 arrangement of 36 characters
+  const regexExp = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi;
```

Comment:
If you cast the string to upper case, you could simplify the regular expression

Reply:
Nice touch Andy. Better for readability of "write-only" regex's.

Reply:
this in mind?

Reply:
Precisely what I was thinking, yeah